devutils/update_submodule.sh

```diff
@@ -0,0 +1,28 @@
+#!/bin/bash
+
+UNGOOGLED_REPO=$(dirname $(dirname $(readlink -f ${BASH_SOURCE[0]})))/ungoogled-chromium
+
+CHROMIUM_VERSION=$(curl -sL https://raw.githubusercontent.com/Eloston/ungoogled-chromium/master/chromium_version.txt)
+UNGOOGLED_REVISION=$(curl -sL https://raw.githubusercontent.com/Eloston/ungoogled-chromium/master/revision.txt)
+
+if [ UNGOOGLED_REVISION != "0" ]
```

Comment:
When will this statement be false (i.e. when is `UNGOOGLED_REVISION` equal to `0`)?

Reply:
That statement will be false whenever `revision.txt` has the number '0' in it.

Reply:
When will `revision.txt` have `0` in it?

Reply:
There's no situation where we ever had to set `0` in revision.txt, and I don't expect there to be a change that will require us to put `0`. You can assume it's always `>= 1`